src/hotspot/share/services/mallocSiteTable.hpp

```diff
@@ -57,12 +57,13 @@ class MallocSite : public AllocationSite {
 class MallocSiteHashtableEntry : public CHeapObj<mtNMT> {
  private:
   MallocSite                         _malloc_site;
+  const unsigned                     _hash;
```

Comment:
Prefer "unsigned int" to be consist with other places. Otherwise, Looks good to me.

Reply:
Thank you Zhengyu!

I reverted to unsigned int. Using just unsigned is a bad habit of mine, sorry.

Cheers, Thomas